### PR TITLE
Updates to use Ansible terminology

### PR DESCRIPTION
**Pull request by governmentSponsored**
Replaces primary computer with control node, client with managed node, and host with managed node. All per https://docs.ansible.com/ansible/latest/network/getting_started/basic_concepts.html.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/linux-install-with-ansible.md [hunk 1]
```diff
@@ -44,13 +44,13 @@ Before you get started, see [the main Defender for Endpoint on Linux page](micro
 
 In addition, for Ansible deployment, you need to be familiar with Ansible administration tasks, have Ansible configured, and know how to deploy playbooks and tasks. Ansible has many ways to complete the same task. These instructions assume availability of supported Ansible modules, such as *apt* and *unarchive* to help deploy the package. Your organization might use a different workflow. Refer to the [Ansible documentation](https://docs.ansible.com/) for details.
 
-- Ansible needs to be installed on at least one computer (we will call it the primary computer).
-- SSH must be configured for an administrator account between the primary computer and all clients, and it is recommended be configured with public key authentication.
-- The following software must be installed on all clients:
+- Ansible needs to be installed on at least one computer (Ansible calls this the control node).
+- SSH must be configured for an administrator account between the control node and all managed nodes (machines that will have Defender installed on them), and it is recommended be configured with public key authentication.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/linux-install-with-ansible.md]:
```suggestion
- SSH must be configured for an administrator account between the control node and all managed nodes (devices that will have Defender for Endpoint installed on them), and it is recommended to be configured with public key authentication.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @governmentSponsored, could you please commit my suggested changes if you agree? Many thanks.

**Comment by governmentSponsored** [comment]
@JohanFreelancer9 that looks great to me. Accepted those changes.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**